pom.xml

```diff
@@ -16,11 +16,11 @@
     <maven.compiler.source>1.8</maven.compiler.source>
 
     <!-- quarkus related -->
-    <quarkus.version>999-SNAPSHOT</quarkus.version>
+    <quarkus.version>1.3.0.Alpha1</quarkus.version>
```

Comment:
shouldn't we keep versions to SNAPSHOT ones ?

Reply:
Hi @radtriste I removed the POM updates now